internal/controller/appconfigurationprovider_controller.go

```diff
@@ -224,6 +224,10 @@ func (reconciler *AzureAppConfigurationProviderReconciler) Reconcile(ctx context
 		retriever = reconciler.Retriever
 	}
 
+	ctx = context.WithValue(ctx, loader.RequestTracingKey, loader.RequestTracing{
+		IsStartUp: reconciler.ProvidersReconcileState[req.NamespacedName].ConfigMapResourceVersion != nil,
```

Comment:
Is ConfigMapResourceVersion `nil` when starting up?

Reply:
My bad.. updated